### PR TITLE
Add some improvements in the documentation

### PR DESCRIPTION
**Pull request by Mingun**

### CODE REVIEW

File: egui/src/ui.rs [hunk 1]
```diff
@@ -1055,7 +1055,7 @@ impl Ui {
     /// Add extra space before the next widget.
     ///
     /// The direction is dependent on the layout.
-    /// This will be in addition to the [`Spacing::item_spacing`}.
+    /// This will be in addition to the [`crate::style::Spacing::item_spacing`].
```
Comment (emilk) [inline egui/src/ui.rs:1058]: Didn't the old link work properly? If so, shouldn't the CI/`check.sh` catch that?

File: epaint/src/shape.rs [hunk 2]
```diff
@@ -15,10 +15,12 @@ pub enum Shape {
     /// For performance reasons it is better to avoid it.
     Vec(Vec<Shape>),
     Circle(CircleShape),
+    /// A line between two points.
     LineSegment {
         points: [Pos2; 2],
         stroke: Stroke,
     },
+    /// A series of lines between points.
```
Comment (emilk) [inline epaint/src/shape.rs:23]:
```suggestion
    /// A series of lines between points.
    /// The path can have a stroke and/or fill (if closed).
```

**Review by emilk — APPROVED** [review]
Thanks!

File: egui/src/ui.rs (hunk 1 above)
Comment (Mingun) [reply egui/src/ui.rs:1058]: No, it looks as a monospaced text in the brackets

File: epaint/src/shape.rs (hunk 2 above)
Comment (Mingun) [reply epaint/src/shape.rs:23]: Added

**Comment by emilk** [comment]
There is a weird merge conflict

**Comment by Mingun** [comment]
Rebased

**Review by emilk — APPROVED** [review]

**Pull request merged**